### DISCUSSION TITLE
Update "zh" language options in the editor

### DISCUSSION THREAD

**Discussion by llnnn252** [godotengine/godot-proposals#9637] [Editor]
The current language options in the editor include **[zh-CN] Chinese, China** and **[zh-TW] Chinese, Taiwan**. However, this classification isn't appropriate because the difference between these two languages lies mainly in their writing systems rather than regions. Moreover, both languages are used beyond just China or Taiwan.

According to the [IETF](https://en.wikipedia.org/wiki/IETF_language_tag#ISO_3166-1_and_UN_M.49), the following changes should be made:
[zh-CN] Chinese, China ➡ [zh-Hans] Chinese (Simplified)
[zh-TW] Chinese, Taiwan ➡ [zh-Hant] Chinese (Traditional)

**Comment by AThousandShips** [comment]
See also:
* https://github.com/godotengine/godot-proposals/issues/6289
